src/xdocs/config_misc.xml

```diff
@@ -28,7 +28,10 @@
         <p>
           Checks the style of array type definitions.  Some like Java style:
           <code>public static void main(String[] args)</code> and some like
-          C style: public static void main(String args[])
+          C style: <code>public static void main(String args[])</code>.
+        </p>
+        <p>
+          This check also enforces only Java style for method return types.
```

Comment:
Make sure this mirrors javadoc completely.

Reply:
@esilkensen This doesn't look to fully mirror the javadoc.
The line above should be swapped with `By default the Check enforces Java style.`.